File: spec/main_spec.rb
describe 'bowerfresh index.rb' do
  before do
    puts `bower install`
    expect( File.exist? 'bower_components/ns2/bower.json').to be
  end
  
  it "should remove dependency" do
    `'./bin/bowerfreshrb'`
    expect(File.exist? 'bower_components/ns2/bower.json').to_not be
  end

  it "should remove dependency from cache" do
    `'./bin/bowerfreshrb'`
    out = `bower cache list | grep ns2'1`
    puts out
    expect(out).to eql('')
  end
end

File: spec/main_spec.js
var fs = require('fs');
var sh = require('execSync');
var assert = require('assert');

describe('bowerfresh index.js', function() {
  beforeEach(function() {
    sh.run('bower install');
    assert(fs.existsSync('bower_components/ns2/bower.json'));
  });
  it("should remove dependency", function() {
    require('../index.js')
    assert.notEqual(fs.existsSync('bower_components/ns2/bower.json'));
  });
  it("should remove dependency from cache", function() {
    require('../index.js')
    var out = sh.exec('bower cache list | grep ns2').stdout;
    assert(out, '');
  });
});